delta-api/pom.xml

```diff
@@ -42,6 +42,12 @@
       <version>6.7.0-SNAPSHOT</version>
       <scope>compile</scope>
     </dependency>
+      <dependency>
+          <groupId>commons-collections</groupId>
+          <artifactId>commons-collections</artifactId>
```

Comment:
My bad, I thought we already had commons collection, let's not add a dependency just for one empty check

Reply:
However CollectionUtils is a basic library which is generally used right ,so can always have basic use cases

Reply:
Yes it's a basic library but we should have good reason to add a dependency as it adds maintenance overhead like upgrades due to security vulnerabilities, I don't think replacing few null checks justifies it